apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -134,5 +141,25 @@ defmodule Aecore.Structures.SpendTx do
     Map.put(account_state, :balance, new_balance)
   end
 
+  @spec is_minimum_fee_met?(SignedTx.t(), :miner | :pool | :validation) :: boolean()
+  def is_minimum_fee_met?(tx, identifier) do
+    if identifier == :validation do
+      true
+    else
+      tx_size_bytes = Pool.get_tx_size_bytes(tx)
+
+      bytes_per_token =
+        case identifier do
+          :pool ->
+            Application.get_env(:aecore, :tx_data)[:pool_fee_bytes_per_token]
+
+          :miner ->
+            Application.get_env(:aecore, :tx_data)[:miner_fee_bytes_per_token]
+        end
+
+      tx.data.fee >= Float.floor(tx_size_bytes / bytes_per_token)
```

Comment:
(very optional) To avoid Floats you can change to `tx.data.fee * bytes_per_token >= tx_size_bytes`